packages/common-sdk/src/web3/transactions/constants.ts

```diff
@@ -0,0 +1,11 @@
+// The hard-coded limit of a transaction size in bytes
+export const TX_LIMIT = 1232;
+
+// The hard-coded limit of an encoded transaction size in bytes
+export const TX_ENCODED_LIMIT = 1644;
```

Comment:
nit: how about ``TX_SIZE_LIMIT`` & ``TX_BASE64_ENCODED_SIZE_LIMIT``

reason1: txnSize() is defined
reason2: LEGACY_TX_UNIQUE_KEYS_LIMIT is defined

is TX_ENCODED_LIMIT considered at the client-side ?

Reply:
`TX_BASE64_ENCODED_SIZE_LIMIT` is used by a class I'm working on that's not yet in the `common-sdk`